content/_data/changelogs/weekly.yml

```diff
@@ -1812,6 +1812,69 @@
     #        message: >
     #          Developer: Validate project names when creating projects using <code>ItemGroupMixIn#createProjectFromXML()</code>
     #          and <code>ItemGroupMixIn#createProject()</code> method
+- version: "2.102"
+  date: 2018-01-14
+  banner: >
+    JEP-200: &quot;Switch Remoting/XStream blacklist to a whitelist&quot; has been integrated to this release.
+    This change implies a HIGH RISK of regressions in plugins.
+    See <a href="/blog/2018/01/13/jep-200/">this blogpost</a> for upgrade guidelines and the list of affected plugins.
+  changes:
+    - type: major rfe
+      pull: 3120
+      message: >
+        Switch Remoting/XStream blacklist to a whitelist.
+      references:
+        - issue: 47736
+        - url: /blog/2018/01/13/jep-200/
+          title: Upgrade guidelines for admins and plugin maintainers
+        - url: https://wiki.jenkins.io/display/JENKINS/Plugins+affected+by+fix+for+JEP-200
+          title: List of affected plugins
+    - type: rfe
+      pull: 3120
+      message: >
+        Update Jenkins Remoting from 3.15 to 3.16.
+      references:
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#316
+          title: full changelog
+        - issue: 47736
+        - issue: 48686
+    - type: rfe
+      pull: 3213
+      issue: 48773
+      message: >
+        Limit which section headers are added to the breadcrumb config outline.
+    - type: rfe
+      pull: 3227
+      message: >
+        Reduce memory consumption for concurrent jobs when retrieving cause of blockage.
+    - type: rfe
+      pull: 3036
+      message: >
+        Introduce new <code>hudson.lifecycle.ExitLifecycle</code> to exit instead of restart.
+      references:
+        - issue: 47043
+        - url: https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties
+          title: Jenkins features controlled by system properties
+    - type: bug
+      pull: 3170
+      issue: 34855
+      message: >
+        <code>AtomicFileWriter</code> was not fully atomic.
+        Now it uses a <code>FileChannel</code> internally and forces a disk sync when committing.
+    - type: rfe
+      pull: 3220
+      message: >
+        Developer: Add a constructor to <code>ToolDescriptor</code> so that subclasses are not required
+        to be an inner class of their <code>Describable</code>.
+    - type: rfe
+      pull: 3222
+      message: >
+        Christmas is over. Revert the Santa Jenkins favicon to a standard one
```

Comment:
We never had a changelog entry for introducing this, so it's weird to have one for its removal.

FWIW I think both should have changelog entries, but it should be consistent.

Reply:
OK, will revert